.gitattributes

```diff
@@ -1 +0,0 @@
-CHANGELOG.asciidoc  merge=union
```

Comment:
I couldn't find a CHANGELOG.asciidoc, so ...

Reply:
Huhn.  That's weird.  Vestigial I guess?

Reply:
It was added here https://github.com/opensearch-project/OpenSearch/commit/3e7fccddaf9c20b3f2349e335bd6733916e3d2ed and then removed here https://github.com/opensearch-project/OpenSearch/commit/0f85c6429cff3f369383d256bc63c05df07bd88c